apps/cowswap-frontend/src/modules/tradeWidgetAddons/containers/TransactionSettings/index.tsx

```diff
@@ -63,9 +64,25 @@ interface TransactionSettingsProps {
   deadlineState: StatefulValue<number>
 }
 
+type SlippageAnalyticsAction = 'Default' | 'Custom'
```

Comment:
The types are identical, might they be generalized? Smth like `TxSettingAction`

Reply:
Addressed